spec/services/convert_deprecated_csharp_parameters_service_spec.rb

```diff
@@ -1,4 +1,6 @@
-describe ConvertDeprecatedCsharpParametersService do
+require "rails_helper"
+
+RSpec.describe ConvertDeprecatedCsharpParametersService do
```

Comment:
`RSpec.describe` vs `describe`

What's the reasoning? I pretty sure I've asked this many a times....

Reply:
https://relishapp.com/rspec/rspec-core/v/3-9/docs/configuration/zero-monkey-patching-mode

We don't but should disable money patching for RSpec. With monkey patching disabled you can't call `describe` and must call `RSpec.describe`